test/main_test.js

```diff
@@ -13,14 +13,12 @@ let testCases = [
     dir: `${__dirname}/fixtures/config_filter`,
     verify: function() {
       let dir = this.dir;
-      return Promise.all(
-        [
-          `${dir}/build.ninja`,
-          `${dir}/random-copy`,
-          `${dir}/one/random-copy`,
-          `${dir}/other/random-copy`
-        ].map(checkExists)
-      );
```

Comment:
Same thing here. If you have stylistic things you want to address, let's move those bits into their own pull request to keep the history clean